packages/email/src/index.ts

```diff
@@ -0,0 +1,72 @@
+import { render } from './render';
+import { theme } from '@csmm/ui';
+
+// cancel
+export async function cancelSubscriptionConfirmation(data: { name: string, undoCancelLink: string }): Promise<string> {
+  const result = render({
+    name: 'generic',
+    data: {
+      name: data.name,
+      paragraph_1: `
+      We are sorry to see you cancel your subscription.
+      You have 30 days left until your data will be removed.
+      `,
+      paragraph_2: 'If this was by accident or if you changed your mind, click the button below.',
+      button: {
+        color: theme.colors.primary,
+        href: data.undoCancelLink,
+        text: 'Undo cancel'
+      }
+    }
+  });
+  return result;
+}
+
+// payment
+export async function paymentConfirmation(data: { name: string, username: string }): Promise<string> {
+  const result = render({
+    name: 'generic',
+    data: {
+      name: data.name,
+      paragraph_1: 'This email confirms your subscription purchase. ',
+      paragraph_2: `The invoice is attached to this email.
+      You can also download a copy on the invoice overview located on your profile.
+      Click the button below to see all your invoices.`,
+      button: {
+        color: theme.colors.info,
+        href: `shop.csmm.app/profile/${data.username}`,
+        text: 'text here'
+      }
+    }
+  });
+  return result;
+}
+
+// deleted
+export async function instanceDeleted(data: { name: string }): Promise<string> {
+  const result = render({
+    name: 'generic',
+    data: {
+      name: data.name,
+      paragraph_1: 'Your instance of CSMM has been deleted. Unfortunately this means your data has been removed and cannot be recovered. You can always resubscribe which will result in a new server.',
```

Comment:
Maybe we want to include some variable here to indicate which subscription/product was deleted exactly?

Reply:
Could indeed add this in the future, should be straight forward